components/events/RequestEvent.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="event-page__section">
```

Comment:
same comment here, this should be `<div class="request-event-section">` or something similar